controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/ApplicationOwnershipConfirmer.java

```diff
@@ -81,6 +82,23 @@ private void ensureConfirmationResponses() {
             });
     }
 
+    private void updateConfirmedApplicationOwners() {
+        ApplicationList.from(controller().applications().asList())
+                .withProjectId()
+                .hasProductionDeployment()
+                .asList()
+                .stream()
+                .filter(application -> application.ownershipIssueId().isPresent())
+                .filter(application -> !application.owner().isPresent())
```

Comment:
This allows the owner to be set only once, i.e., to never change. I think we should allow that.